rafx-framework/src/visibility/visibility_region.rs

```diff
@@ -0,0 +1,104 @@
+use crate::visibility::view_frustum_object::ViewFrustumObjectHandle;
+use crate::visibility::visibility_object::{CullModel, VisibilityObjectHandle};
+use crate::visibility::visibility_object_allocator::{
+    VisibilityObjectAllocator, VisibilityObjectId, VisibilityObjectRef,
+};
+use crate::visibility::EntityId;
+use crossbeam_channel::Sender;
+use rafx_visibility::{AsyncCommand, VisibilityWorldArc, ZoneHandle};
+use std::sync::Arc;
+
+pub struct VisibilityRegion {
+    allocator: Arc<VisibilityObjectAllocator>,
+    commands: Sender<AsyncCommand>,
+    dynamic_zone: ZoneHandle,
+    static_zone: ZoneHandle,
+}
+
+impl VisibilityRegion {
+    pub fn new() -> Self {
+        let visibility_world = VisibilityWorldArc::new();
+        let allocator = Arc::new(VisibilityObjectAllocator::new(visibility_world.clone()));
+
+        let static_zone = {
+            let mut inner = visibility_world.inner.lock();
```

Comment:
very minor thing but I imagine we could only lock once here, or not lock at all if we defer putting visibility_world into an Arc until after we finish populating it